util-security/src/main/scala/com/twitter/util/security/X500PrincipalInfo.scala

```diff
@@ -31,8 +31,9 @@ private object X500PrincipalInfo {
    */
   private def parseNameToMap(name: String): Map[String, String] = {
     val ldapName: LdapName = new LdapName(name)
-    val rdns: List[Rdn] = ldapName.getRdns().toList
+    val rdns: List[Rdn] = ldapName.getRdns().asScala.toList
     rdns.map(item => (item.getType, item.getValue.toString)).toMap
   }
 
 }
+n
```

Comment:
Oh, sorry... 😨 
compilation failed because of this unintended `n`.
I'll fix it 🙇